### PR TITLE
updated react-leaflet dependency version

### PR DESCRIPTION
**Pull request by kenzoarima**
npm install was previously failing. Found [this advice](https://github.com/PaulLeCam/react-leaflet/issues/887), tried it, and it worked 💪

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "location-geo-map",
-  "version": "2.4.7",
+  "version": "2.4.7.1",
```
Comment (jsbnr) [inline package.json]: onlt three components are allowed, so we can bump to 2.4.8 (or whatever)

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "prop-types": "15.7.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-leaflet": "^2.7.0",
+    "react-leaflet": "^2.8.0",
```
Comment (jsbnr) [inline package.json:15]: Seems to work ok from my test, im still not quite sure why this bump was helpful for you.

Comment (jsbnr) [reply package.json:15]: ah i see now, it allows you to avoid the peer dependencies problem

**Comment by kenzoarima** [comment]
@jsbnr sir, I think the CLA issue is fixed now.

**Pull request merged**